### ISSUE TITLE
High definition Luna icons

### ISSUE THREAD

**Issue by SelfRef** [rozniak/xfce-winxp-tc#141]
I see the icons this project uses are original Luna icons that are up to 48x48 in size. There is a very recent pack of icons from one guy that have recreated all stock Luna icons in the size of 1024x1024.

While bigger sizes may be not required to mimic the original XP look, they would certainly make sense on higher DPI screens, docks and generally would be nice to have them included.

[DeviantArt link](https://www.deviantart.com/marchmountain/art/Windows-XP-High-Resolution-Icon-Pack-916042853)

**Comment by rozniak** [comment]
I have seen these, the thing is I'm not sure how well it'll work if you have these icons that are super high-res, but then things like `.c` source code file icons are like billy basic 32x32 still. At least with original icons across the board, they're consistent in that regard (they look equally crappy at higher res - sort of I guess with some being 48x48 but it's close enough).

Would have to have a think about it, also would appreciate any comments anyone else might have regarding the above. Also would need to get in touch and ask permission if deciding to include them.

**Comment by daemonspudguy** [comment]
I know a project with high quality XP icons. [This](https://github.com/B00merang-Artwork/Windows-XP/) is an icon pack by the B00merang Project based on the old Ylmf-OS icons.

**Comment by BoQsc** [comment]
> Also would need to get in touch and ask permission if deciding to include them.

Few minutes ago, I've sent him a message. We'll see how it goes.
![image](https://user-images.githubusercontent.com/21064622/173771894-4b85fa87-7475-4842-880d-77ed9e2e45fa.png)

**Comment by BoQsc** [comment]
![image](https://user-images.githubusercontent.com/21064622/174395627-cf73d749-c386-4e09-bb4f-82b04728aa44.png)
### Update | The permissions were granted by March Mountain
**March Mountain** was very welcoming and released his icons into Public Domain.
https://github.com/marchmountain/-Windows-XP-High-Resolution-Icon-Pack/blob/main/LICENSE

This means [xfce-winxp-tc ](https://github.com/rozniak/xfce-winxp-tc) project now can use these amazing re-drawn icons of `1024x1024` resolution.

Thanks to the [**March Mountain**](https://github.com/marchmountain/) for being this amazing.

His DeviantART profile is here: 
https://www.deviantart.com/marchmountain

Source files [.psd]
---
The **.psd** source files are available here and requires **Adobe CS2** to generate the final Icons.
https://github.com/marchmountain/-Windows-XP-High-Resolution-Icon-Pack/releases/tag/01

**License:** .zip archive contains [CC0 Creative Commons Zero (Public Domain) License.](https://creativecommons.org/publicdomain/zero/1.0/)
**Tip:**  (Office 2003, if any other files encountered in the source archive)

**Download [.png]**
---
The following release is also subject to the Creative Commons Zero License. (Public Domain)
https://www.deviantart.com/marchmountain/art/Windows-XP-High-Resolution-Icon-Pack-916042853


![image](https://user-images.githubusercontent.com/21064622/174395299-9e9ef6a7-ee39-4b6c-9cf1-4bf9a331ada3.png)

**Comment by SelfRef** [comment]
Wow, I was hoping to convince the author to agree for use in this project but making them CC0 is much better idea. Big thanks @BoQsc and @marchmountain :smile:

**Comment by Techwizz-somboo** [comment]
@rozniak what's you're current thoughts on potentially including these icons?

**Comment by rozniak** [comment]
I'm not sure at the moment, I suppose I tend to prefer sticking to original sources as much as possible. Though could include them as an optional target? (eg. under `/icons/luna-hires` or something?)

**Comment by Techwizz-somboo** [comment]
> I'm not sure at the moment, I suppose I tend to prefer sticking to original sources as much as possible. Though could include them as an optional target? (eg. under `/icons/luna-hires` or something?)

I like that idea! I think going for something like that would make a lot of people happy.

**Comment by rozniak** [comment]
I'll have a look at some point... I think there's still icons that need doing. Also some Windows 2000 / Watercolor icons. I suppose I say "I'll look into it" for basically everything :grin: 

(supposed to be a grin react, not a laugh react on your comment - I thought there used to be custom emote reacts on here? :raised_eyebrow: :eyes: )

**Comment by MrMegarich** [comment]
Yes keep it optional, some people might like to keep the old icons.
